hasjob/templates/confirm_email.html

```diff
@@ -0,0 +1,28 @@
+<div itemscope itemtype="http://schema.org/EmailMessage">
```

Comment:
Coding style for HTML requires two space indentation for nested tags and four space when a long tag is wrapped (which is rare).